File: dart/2685.dart
import 'dart:io';

void main() {
  while (true) {
    var inp = stdin.readLineSync();

    if (inp == null) {
      break;
    }

    int n = int.parse(inp);

    if ((n >= 0 && n < 90) || n == 360) {
      print('Bom Dia!!');
    } else if (n >= 90 && n < 180) {
      print('Boa Tarde!!');
    } else if (n >= 180 && n < 270) {
      print('Boa Noite!!');
    } else if (n >= 270 && n < 360) {
      print('De Madrugada!!');
    }
  }
}

File: python/2685.py
while True:
    try:
        ang = int(input())

        if (ang >= 0 and ang < 90) or ang == 360:
            print('Bom Dia!!')
        if ang >= 90 and ang < 180:
            print('Boa Tarde!!')
        if ang >= 180 and ang < 270:
            print('Boa Noite!!')
        if ang >= 270 and ang < 360:
            print('De Madrugada!!')
    except EOFError:
        break
